MAUI/DataGrid/Column-types.md

```diff
@@ -366,6 +366,21 @@ private void dataGrid_AutoGeneratingColumn(object sender, DataGridAutoGenerating
 {% endhighlight %}
 {% endtabs %}
 
+### Header Line break mode
+
+The text wrapping and truncation of a column's header text can be customized by setting the DataGridColumn.HeaderLineBreakMode property.
+ 
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid AutoGenerateColumnsMode="None"
+                       ItemsSource="{Binding OrderInfo}">
+    <syncfusion:SfDataGrid.Columns >
+        <syncfusion:DataGridTextColumn MappingName="Name" HeaderLineBreakMode="NoWrap" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
```

Comment:
Could you include the note here also?

N> The truncation modes will not work on the Windows platform.